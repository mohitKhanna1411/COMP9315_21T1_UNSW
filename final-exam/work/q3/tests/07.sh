./bs 60 + -4 -55 -50 -9 -45 -14 -40 -19 -35 -24 -30
