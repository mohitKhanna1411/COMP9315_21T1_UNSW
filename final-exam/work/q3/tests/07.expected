init: 111111111111111111111111111111111111111111111111111111111111
-004: 111111111111111111111111111111111111111111111111111111101111
-055: 111101111111111111111111111111111111111111111111111111101111
-050: 111101111011111111111111111111111111111111111111111111101111
-009: 111101111011111111111111111111111111111111111111110111101111
-045: 111101111011110111111111111111111111111111111111110111101111
-014: 111101111011110111111111111111111111111111111011110111101111
-040: 111101111011110111101111111111111111111111111011110111101111
-019: 111101111011110111101111111111111111111101111011110111101111
-035: 111101111011110111101111011111111111111101111011110111101111
-024: 111101111011110111101111011111111110111101111011110111101111
-030: 111101111011110111101111011110111110111101111011110111101111
